components/homePage/Banner.tsx

```diff
@@ -96,6 +119,70 @@ const Banner = () => {
               </Box>
               {bannerDetails.bannercta2Caption}
             </Button>
+      {/* modal display the success */}
+          <Modal
+          keepMounted
+          open={open}
+          onClose={handleClose}
+          aria-labelledby="keep-mounted-modal-title"
+          aria-describedby="keep-mounted-modal-description"
+
+      >
+          <Box sx={style}> 
+          <DialogContent
+          dividers
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            backgroundColor: 'lightgreen',
+            borderRadius:"10px",
+          }}
+        >
+          <Image 
+          src={image1} alt="test" style={{ maxWidth: '25%', height: 'auto',}} />
+        </DialogContent>       
+          
+        <DialogContent
+          dividers
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            textAlign: 'center',
+          }}
+        >
+          <Typography id="keep-mounted-modal-title" variant="h6" component="h2" sx={{color: "#54BC2F",}}>
+          {bannerDetails.modalTitle}
+          </Typography>
+          
+          <Typography id="keep-mounted-modal-description" sx={{ mt: 2, }}>
+          {bannerDetails.modalContentText}
+          </Typography>
+          <Button
+              variant="contained"
+              color="success"
+              autoFocus
+              onClick={handleClose}
+              sx={{
+                backgroundColor: "#52AA44",
+                width: "30px",
+                height: "30px",
+                borderRadius: "58px",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+                marginTop: "20px",
+              }}
+
+            >
+              Ok
```

Comment:
This button caption should also be configurable.

Reply:
**got it**